docs/howto/graph-api/links.md

```diff
@@ -0,0 +1,131 @@
+# The Graph Part
+
+In the previous page, we explained how to fetch data from existing data structures.
+This along does not make it a 'graph' because we only have isolated 'nodes'.
+To navigate from one data structure to another, we need to introduce 'edges' to link 'nodes' together.
+
+## Graph Edges
+
+The data structures in `NOMAD` are inherently linked together.
+For example, an upload is a collection of entries, an entry corresponding to an archive, a user group is a collection of users,
+each user may be the owner of several uploads, etc.
+
+Thus, it is natural to link these data structures together via **special tokens**.
+The following figure illustrates the relationships between the data structures in `NOMAD` with the special tokens highlighted.
+
+![NOMAD graph](graph.svg)
+
+For example, if there is an upload with ID `example_upload_id`, and it has an entry with ID `example_entry_id`,
+the request to fetch the `entry_id`and `entry_create_time` of the entry, together with `upload_create_time` in the upload, would look like this.
+
+```json title="fetching linked data" hl_lines="5"
+{
+  "uploads": {
+    "example_upload_id": {
+      "upload_create_time": {"m_request": {"directive": "plain"}},
+      "entries": {
+        "example_entry_id": {
+          "entry_id": {"m_request": {"directive": "plain"}},
+          "entry_create_time": {"m_request": {"directive": "plain"}}
+        }
+      }
+    }
+  }
+}
+```
```

Comment:
This belongs to the "Basics" page: this is a key concept and should be introduced in the basics. Just put it at the end of the page.

Reply:
It better stays here.